libs/viewer/src/Settings_generated.cpp

```diff
@@ -350,7 +350,7 @@ int parse(jsmntok_t const* tokens, int i, const char* jsonChunk, FogOptions* out
 std::ostream& operator<<(std::ostream& out, const FogOptions& in) {
     return out << "{\n"
         << "\"distance\": " << (in.distance) << ",\n"
-        << "\"cutOffDistance\": " << (in.cutOffDistance) << ",\n"
+        << "\"cutOffDistance\": " << (std::isinf(in.cutOffDistance) ? "\"INFINITY\"" : std::to_string(in.cutOffDistance)) << ",\n"
```

Comment:
Technically, this is not valid JSON, which in and of itself isn't a problem, but it might require adjusting our parser's code. Does the import script handle `inf` like this?

Reply:
the import script can't hadle `inf` or `nan`, because `load()` fails.

Reply:
There's a hacky way of fixing this, by writing a large number (like `1e9999`) instead of a string, which might work with our parser.